### PR TITLE
Intro

### PR DESCRIPTION
**Pull request by osdiab**

### CODE REVIEW

File: report.md [hunk 1]
```diff
@@ -5,3 +5,114 @@
...
+like this project.
+
+The WebRTC protocol provides three APIs:
+
+1. `MediaStream` (get camera and microphone data, accessed via `navigator.getUserMedia`)
+2. `PeerConnection` (sending and receiving media)
+3. `DataChannel` (sending arbitrary data directly between browsers)
+
+For this project, while we are not accessing camera and microphone data, the MediaStream
+abstraction can be used with media sourced on local disks as well, with a little work; and the
```
Comment (markulrich) [inline report.md:80]: We only use the DataChannel

File: report.md [hunk 2]
```diff
@@ -5,3 +5,114 @@
...
+
+1. No specialized hardware. It relies entirely on computing devices the average consumer would
+already have in their home, like mobile phones, tablets, laptops, or desktops.
+2. No software setup. One can just go to a website in a browser on any computing devices they have,
+and without any software installation, can synchronize music playback and selection. This is
+possible due to recent developments in Web Real Time Communication (WebRTC), which will be
+discussed in the next section.
+3. Flexibility: Nodes can be added or removed just by accessing or navigating away from a URL in a
+browser. Once again, this requires no technical expertise or setup.
+4. Expanded media library selection. Each device can have a different set of music files to
```
Comment (markulrich) [inline report.md:51]: I think major downside of our technique is that all media files are in temporary storage, this should be mentioned. At the moment, it's probably better suited to impromptu gathers like if you're at the beach and one person's phone isn't loud enough so you want to play on everyone's phone

Comment (osdiab) [inline report.md:51]: 40db6d6

File: report.md (hunk 1 above)
Comment (osdiab) [inline report.md:80]: 41f6114

**Pull request closed without merging**